README.md

```diff
@@ -110,36 +110,71 @@ class Inspire
 }
 ```
 
-### Laravel Authentication
-A Laravel user provider is also included to configure Laravel's built-in authentication to validate against Northstar
-instead of your local database. After configuring the client above, set your application to use the `northstar` driver
-instead of `eloquent` in `config/auth.php`.
+### Authentication
+You can use the `authorize` and `logout` methods on the client to let users log in using Northstar's single-sign on
+functionality. This can be implemented anywhere using a custom framework bridge, but it's super easy in Laravel:
 
-```php
-// For Laravel 5.0 or 5.1
-'driver' => 'northstar',
-'model' => App\User::class,
+First, set up the `login` and `logout` routes in your `routes.php`:
 
-// For Laravel 5.2+
-'providers' => [
-    'users' => [
-        'driver' => 'northstar',
-        'model' => App\User::class,
-    ],
-    // ...
- ]
+```php
+// Authentication
+Router::get('login', 'AuthController@getLogin');
+Router::get('logout', 'AuthController@getLogout');
 ```
 
-Make sure to switch the Northstar client to use the password grant in `config/services.php`:
+And forward those requests to the Northstar client in your authentication controller:
 
 ```php
-    'northstar' => [
-        'grant' => 'password',
-        // ...
-    ]
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+class AuthController extends Controller
+{
+    /**
+     * Where to redirect users after login.
+     *
+     * @var string
+     */
+    protected $redirectTo = '/users';
+
+    /**
+     * Where to redirect users after logout.
+     *
+     * @var string
+     */
+    protected $redirectAfterLogout = '/';
+
+    /**
+     * Handle a login request to the application.
+     *
+     * @param ServerRequestInterface $request
+     * @param ResponseInterface $response
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function getLogin(ServerRequestInterface $request, ResponseInterface $response)
+    {
+        return Northstar::authorize($request, $response, $this->redirectTo);
```

Comment:
Agh yeah, it's using the facade there (I know, I know 😆) but it's confusing. I'd done it so the docs didn't have all the extra boilerplate of injecting the class. I replaced it with `app('northstar')` in 678c554.